README.md

```diff
@@ -22,34 +22,71 @@ A hosted [Hugging Face space](https://huggingface.co/spaces/sesame/csm-1b) is al
 
 ### Setup
 
+Clone and setup the repo:
+
 ```bash
 git clone git@github.com:SesameAILabs/csm.git
 cd csm
 python3.10 -m venv .venv
 source .venv/bin/activate
+
+# Set environment variable to disable Triton compilation
+export NO_TORCH_COMPILE=1
+
 pip install -r requirements.txt
 ```
 
-### Windows Setup
+Install ffmpeg (required for audio processing):
+
+```bash
+# On macOS with Homebrew
+brew install ffmpeg
+
+# On Ubuntu/Debian
+sudo apt-get install ffmpeg
+
+# On Windows
+# Download from https://ffmpeg.org/download.html
+```
+
+Download the required prompt audio files:
+
+```bash
+# Create prompts directory
+mkdir -p prompts
+
+# Download prompt files and place them in the prompts directory
+https://huggingface.co/spaces/sesame/csm-1b/tree/main/prompts
+```
```

Comment:
Remove this once the run.py is automatically pulling from HF repo, see [this comment](https://github.com/SesameAILabs/csm/pull/66/files#r1996787526)